### DISCUSSION TITLE
Where can I find the vm.args

### DISCUSSION THREAD

**Discussion by dimer191996** [apache/couchdb#3680] [Q&A]
I'm new to couchdb and i have this error
```
This database failed to load.
Error running query. Reason: (nodedown) progress not possible
```

* CouchDB version used: new one
* Browser name and version: chrome. new one
* Operating system and version:windows

**Comment by wohali** [comment]
If you are looking for general support with using CouchDB, and a reply does not come here quickly, please try the Slack/IRC chat room. Joining instructions are [here](http://couchdb.apache.org/#chat)
